packages/cli/tests/e2e/bot/CommandExecutionIsUserError.ts

```diff
@@ -29,7 +29,7 @@ describe("Error type should be expected", function () {
   const env = environmentManager.getDefaultEnvName();
 
   after(async () => {
-    await cleanUp(appName, projectPath, true, true, false, true);
+    await cleanUp(appName, projectPath, true, true, false, "dev");
```

Comment:
Good catch! since there's a variable `env` above, how about using the variable instead?

Reply:
done